pkg/controller/operators/catalog/operator.go

```diff
@@ -1726,22 +1726,34 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 					Resource: r.Name,
 				}
 
-				// Set up the dynamic client ResourceInterface
+				owner := &v1alpha1.ClusterServiceVersion{}
+				owner.SetNamespace(plan.GetNamespace())
+				owner.SetName(step.Resolving)
```

Comment:
Does `step.Resolving` always point to the CSV name?

Reply:
It should for all InstallPlans generated by OLM -- but you're right, it could be more robust. I'll go ahead and add some bumpers.